### PR TITLE
fix icon in notification, adds a customizable subject prefix

### PR DESCRIPTION
**Pull request by blizzz**
This is when you get your notification:

image

This is the configured notification

Screenshot_20200806_011736

and just with the placeholder:

Screenshot_20200806_011811

### CODE REVIEW

File: src/views/FlowNotify.vue [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<template>
+	<div>
+		<input type="text"
+			max-length="80"
+			:placeholder=placeholder
+			:value=currentInscription
+			@input="(newValue) => newValue !== null && $emit('input', JSON.stringify({ inscription: newValue.target.value } ))" />
```
Comment (juliushaertl) [inline src/views/FlowNotify.vue:7]: Would be good to move that to a dedicated method.

File: stylelint.config.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+module.exports = {
```
Comment (juliushaertl) [inline stylelint.config.js:1]: @skjnldsv I guess we should also move this to a `@nextcloud` package.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    "sass-loader": "^8.0.2",
+    "stylelint": "^13.6.1",
+    "stylelint-config-recommended-scss": "^4.2.0",
+    "stylelint-scss": "^3.18.0",
+    "stylelint-webpack-plugin": "^2.1.0",
+    "vue-loader": "^15.9.3",
+    "vue-template-compiler": "^2.6.11",
+    "webpack": "^4.44.1",
+    "webpack-cli": "^3.3.12",
+    "webpack-merge": "^5.1.1"
```
Comment (juliushaertl) [inline package.json:53]: This is not needed, as the default webpack config is used without modification.

File: stylelint.config.js (hunk 2 above)
Comment (skjnldsv) [reply stylelint.config.js:1]: Yes, there is none yet :shrug:

File: package.json (hunk 3 above)
Comment (blizzz) [reply package.json:53]: fwiw i was cherry-picking from https://github.com/nextcloud/app-tutorial 😇

File: src/views/FlowNotify.vue (hunk 1 above)
Comment (blizzz) [reply src/views/FlowNotify.vue]: addressed

File: package.json (hunk 3 above)
Comment (blizzz) [reply package.json]: addressed

**Pull request merged**